extensions/sql/processors/ExecuteSQL.cpp

```diff
@@ -81,14 +81,14 @@ void ExecuteSQL::initialize() {
   setSupportedRelationships( { s_success });
 }
 
-void ExecuteSQL::processOnSchedule(const core::ProcessContext &context) {
+void ExecuteSQL::processOnSchedule(core::ProcessContext &context) {
   initOutputFormat(context);
 
   context.getProperty(s_sqlSelectQuery.getName(), sqlSelectQuery_);
   context.getProperty(s_maxRowsPerFlowFile.getName(), max_rows_);
 }
 
-void ExecuteSQL::processOnTrigger(core::ProcessSession &session) {
+void ExecuteSQL::processOnTrigger(core::ProcessContext& /*context*/, core::ProcessSession& session) {
```

Comment:
In a previous iteration QueryDatabaseTables's processOnTrigger needed the context, so it had to be put on the CRTP interface of SQLProcessor. It has been rewritten since and it is no longer needed by QDT, so I removed it.